src/metabase/driver/postgres/actions.clj

```diff
@@ -3,52 +3,58 @@
   (:require
    [clojure.java.jdbc :as jdbc]
    [metabase.driver.sql-jdbc.actions :as sql-jdbc.actions]
+   [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
+   [metabase.util :as u]
    [metabase.util.i18n :refer [tru]]))
 
-(defn- constraint->columns [conn constraint-name]
```

Comment:
I moved this down a bit so it appears directly above the function where it's used for the first time.